security/pkg/stsservice/tokenmanager/tokenmanager.go

```diff
@@ -53,7 +53,7 @@ type GCPProjectInfo struct {
 	clusterLocation string
 }
 
-func getGCPProjectInfo() GCPProjectInfo {
+func GetGCPProjectInfo() GCPProjectInfo {
```

Comment:
Is there a reason we expose this function to public?

Reply:
Yes, it will be used to avoid the complex setup for ManagedCA - where we need operator to add GKE_URL as metadata to proxy config, etc.

Stackdriver integration already has all the info. Simpler is better.

Reply:
nice